src/ZendeskApi.Client/Resources/Article/ArticlesResources.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using ZendeskApi.Client.Models;
+using ZendeskApi.Client.Resources.Interfaces;
+using ZendeskApi.Client.Extensions;
+using ZendeskApi.Client.Exceptions;
+using ZendeskApi.Client.Responses.Articles;
+using ZendeskApi.Client.Responses;
+
+namespace ZendeskApi.Client.Resources.Articles
+{
+    public class ArticlesResources : IArticlesResources
+    {
+        private const string ResourceUri = "api/v2/help_center/articles/";
+
+        private readonly IZendeskApiClient _apiClient;
+        private readonly ILogger _logger;
+
+        private readonly Func<ILogger, string, IDisposable> _loggerScope = LoggerMessage.DefineScope<string>(typeof(ArticlesResources).Name + ": {0}");
+
+        public ArticlesResources(IZendeskApiClient apiClient, ILogger logger)
+        {
+            _apiClient = apiClient;
+            _logger = logger;
+        }
+
+
+        public async System.Threading.Tasks.Task<Responses.IPagination<Article>> ListAsync(string query, PagerParameters pager = null)
```

Comment:
Can we add the `using`'s for this instead, and just have it as `async Task<IPagination<Article>>`.

Would you mind also changing it's name to 'GetAsync'? This is the new convention we're trying to adhere to, and we're slowly rolling it out throughout resources.